src/app/components/authorized/post/post.component.ts

```diff
@@ -10,6 +12,8 @@ import { Post } from 'src/types/posts';
 export class PostComponent implements OnInit {
     @Input() postId: number;
 
+    posts$: Post[];
+
```

Comment:
why?

Reply:
Open questions should not be in lead review. This should not be approved by you and should be answered.